hypershift/baremetal/agent.md

```diff
@@ -0,0 +1,565 @@
+# **HyperShift + Bare Metal workers using 'agent' provider**
+
+This document explains how to deploy HyperShift in OpenShift and then use the 'agent' provider to deploy bare metal worker nodes for our `HostedClusters`.
+
+This work tries to answer the questions exposed in [this card](https://issues.redhat.com/browse/KNIECO-4458).
+
+> **WARNING**: HyperShift is a work-in-progress project and it is not officially supported by Red Hat.
+
+## **Hypershift Operator requirements**
+
+* cluster-admin access to an OpenShift Cluster (We tested it with 4.9.21) to deploy the CRDs + operator. See the [ocp-bm-cluster](../assets/ocp-bm-cluster.md) document on how to deploy a baremetal OCP cluster using kcli with virtual masters
+* 1 filesystem type Persistent Volume (4Gi minimum) to store the `etcd` database for demo purposes (3x for 'production' environments)
+* Assisted Service and Hive already installed. See the [assisted-service-hive](../assets/assisted-service-hive.md) document on how to do it
```

Comment:
Maybe add a note saying that you also get these components when using RHACM 2.4+

Reply:
The thing is... I had some issues in the past if using RHACM (some RBAC stuff between HyperShift and RHACM)... that's why I used only the required bits.